exporter/elasticsearchexporter/model.go

```diff
@@ -57,6 +57,11 @@ var resourceAttrsConversionMap = map[string]string{
 	semconv.AttributeK8SNodeName:            "kubernetes.node.name",
 	semconv.AttributeK8SPodName:             "kubernetes.pod.name",
 	semconv.AttributeK8SPodUID:              "kubernetes.pod.uid",
+	semconv.AttributeK8SJobName:             "kubernetes.job.name",
+	semconv.AttributeK8SCronJobName:         "kubernetes.cronjob.name",
+	semconv.AttributeK8SStatefulSetName:     "kubernetes.statefulset.name",
+	semconv.AttributeK8SReplicaSetName:      "kubernetes.replicaset.name",
+	semconv.AttributeK8SDaemonSetName:       "kubernetes.daemonset.name",
```

Comment:
Shall we also map the `k8s.container.name` too? 

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/528678e607b60b00cbb5432b7ba8d2fcdc8cb810/processor/k8sattributesprocessor/config.go#L146

Reply:
added cluster name and container name for consistency.